Article/Article.js

```diff
@@ -1,29 +1,51 @@
-// Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+'use strict'
+
+/**
+ * Constants
+ */
+
+const articles = Array.from(document.getElementsByClassName('article'))
+
+/**
+ * Define class
+ */
 
 class Article {
   constructor(domElement) {
-    // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.domElement = domElement
+    this.isExpanded = false
+    this.expandButton = this.domElement.querySelector('.expandButton')
+    this.expandButton.textContent = 'expand'
+    this.expandButton.addEventListener(
+      'click', () => this.isExpanded ? this.closeArticle() : this.expandArticle()
+    )
   }
 
   expandArticle() {
-    // Using our reference to the domElement, toggle a class to expand or hide the article.
+    this.isExpanded = true
+    this.domElement.classList.add('article-open')
+    this.domElement.classList.remove('close')
+    this.expandButton.textContent = 'close'
+  }
 
+  closeArticle() {
+    this.isExpanded = false
+    this.domElement.classList.add('close')
+    this.domElement.classList.remove('article-open')
+    this.expandButton.textContent = 'expand'
   }
```

Comment:
That's cool you were able to complete the objective without using the `toggle` classList method